src/pivot-table/components/DimensionCell.tsx

```diff
@@ -1,26 +1,30 @@
 import React from 'react';
-import { Pressable, StyleSheet, View, Text, ViewStyle } from "react-native";
 import { Model } from '../../types/types';
 import { NxPivotDimensionCell } from '../../types/QIX';
 import { Cell } from '../handle-data';
-import sharedStyles from './shared-styles';
+import { borderStyle, textStyle } from './shared-styles';
 
 export interface DimensionCellProps {
   cell: Cell;
   model: Model;
   rowIndex: number;
   colIndex: number;
   isLeftColumn?: boolean;
-  style: ViewStyle | ViewStyle[];
+  style: any;
```

Comment:
We should type this as well. Does this contain all the styling for a Cell?

Reply:
It should be typed yes. It only contains the absolute position of the cell. Which is coming from the `react-window` component.